pkg/cloudclient/gcp/private.go

```diff
@@ -0,0 +1,433 @@
+package gcp
+
+//Features to add - image-id, kms-key-id
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"os"
+	"regexp"
+	"time"
+	// "path/filepath"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	"golang.org/x/oauth2/google"
+	computev1 "google.golang.org/api/compute/v1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+
+	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
+	handledErrors "github.com/openshift/osd-network-verifier/pkg/errors"
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+type createE2InstanceInput struct {
```

Comment:
General comment for this file: Some of the things named after amazon specific services can be named after gcp specific ones in this file. e.g. createEC2Instance can be createComputeServiceInstance, validateInstanceType can be validateMachineType, AMI can be  container optimized image. That way we can stay consistent with GCP's sdk references to these objects, just like we do with Amazon. I'd reference the GCP libraries we've imported in vendor/ dir for standards.

Reply:
Updated method names. Is it fine now? By "I'd reference the GCP libraries we've imported in vendor/ dir for standards.", do you mean referencing library url within the method body?